src/js/electricPianoBand.js

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("floe.chartAuthoring.electricPianoBand", {
```

Comment:
What would probably be helpful at some point (not urgent) would be to annotate this file so it can be used as more of a model synth definition for those largely unfamiliar with how Flocking works or with the particularities of audio synthesis to create their own synths. Maybe @colinbdclark & I can do this together at some point.

I could eventually see something along the lines of an approach to generating the synth definitions Flocking needs from a simple user interface that has pre-defined instrument definitions, etc etc. I'm thinking waaaay back to when I used to play around with MIDI files in the early 90s - just changing the instruments used in MIDI files I found online to hear what things sounded like on a tuba vs. a piano, etc etc.